openedx/core/djangoapps/cors_csrf/tests/test_middleware.py

```diff
@@ -86,25 +97,63 @@ def test_disabled_no_cors_headers(self):
             CorsCSRFMiddleware(get_response=lambda request: None)
 
     @override_settings(CORS_ORIGIN_WHITELIST=['https://bar.com'])
-    def test_disabled_wrong_cors_domain(self):
+    @mock.patch('openedx.core.djangoapps.cors_csrf.middleware.set_custom_attribute')
+    def test_disabled_wrong_cors_domain(self, mock_set_custom_attribute):
```

Comment:
In theory you'd want to test that the toggle works for a variety of tests. One way that I've done this is to use `@ddt.data(True, False)` for the toggle, as can be seen in this example PR: https://github.com/openedx/auth-backends/pull/402/files#diff-55b84b158e4b1de1692b4813bbe57a981a678c3e31bb262253af17382edcb48e.

Consider adding for some tests as appropriate.